docs/install/ha.md

```diff
@@ -70,16 +71,19 @@ tls-san:
   - another-kubernetes-domain.com
 
 ```
+
 As mentioned previously, you must have an odd number of server nodes in total.
 
 ### 4. Confirm cluster is functional
 Once you've launched the `rke2 server` process on all server nodes, ensure that the cluster has come up properly with
+
 ```bash
+# FOR PR: running kubectl as root only because of  /etc/rancher/rke2/rke2.yaml read access sounds like overkill, should it pointed out?
```

Comment:
This might be outside of the initial scope. But, should we tell the user that they don't need root access for this step if:

1. kubectl is already installed
2. kubectl context has been configured for this new cluster

Reply:
A reference to this might be helpful. https://docs.rke2.io/cluster_access/#cluster-access

Also, a config file can be placed on disk with the content below that won't require `root`.

```yaml
write-kubeconfig-mode: 644
```